### PR TITLE
Merge Nessus and Nessus WAS into single Tenable parser

### LINKED ISSUE #7477: CVSS import error while importing vulnerabilities from Nessus
**Be informative**
While trying to import vulnerability scan data from Nessus, seeing an error for CVSS import field

**Bug description**
An exception error occurred during the report import:Malformed CVSS3 vector "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N" is missing mandatory prefix or uses unsupported CVSS version

**Steps to reproduce**
1. Export scan data from Nessus
2. Create a new Engagment and Test
3. Import scan data
4. An error is received on the UI

**Deployment method** *(select with an `X`)*
- [X] Docker Compose
- [ ] Kubernetes
- [ ] GoDojo

**Environment information**
 - Operating System: [e.g. Ubuntu 20.04]
 - DefectDojo version 2.18.2

**Logs** 
[23/Jan/2023 14:23:22] ERROR [dojo.test.views:754] Malformed CVSS3 vector "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N" is missing mandatory prefix or uses unsupported CVSS version
Traceback (most recent call last):
  File "/app/dojo/test/views.py", line 745, in re_import_scan_results
    reimporter.reimport_scan(scan, scan_type, test, active=active, verified=verified,
  File "/app/dojo/importers/reimporter/reimporter.py", line 383, in reimport_scan
    parsed_findings = parser.get_findings(scan, test)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/dojo/tools/nessus/parser.py", line 264, in get_findings
    return NessusXMLParser().get_findings(filename, test)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/dojo/tools/nessus/parser.py", line 197, in get_findings
    cvssv3 = CVSS3(item.findtext("cvss3_vector")).clean_vector()
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/cvss/cvss3.py", line 96, in __init__
    self.parse_vector()
  File "/usr/local/lib/python3.11/site-packages/cvss/cvss3.py", line 123, in parse_vector
    raise CVSS3MalformedError('Malformed CVSS3 vector "{0}" is missing mandatory prefix '
cvss.exceptions.CVSS3MalformedError: Malformed CVSS3 vector "AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:N" is missing mandatory prefix or uses unsupported CVSS version

**Screenshots**
image

### PR DESCRIPTION
**Pull request by Maffooch**
There has been some unification of the Nessus and Nessus WAS tools within Tenable such that it is confusing which parser should be used. This effort is a way to simplify this process by combining the Nessus and Nessus WAS parsers into a single Tenable parser. The strategy is to merge the CSV and XML parsers from each parser into a single parser that accommodates fields from both reports if they are present

- [x] Unification of parsers
- [x] Unification of unit tests
- [x] Migration to move Nessus/NessusWAS test types to Tenable test types on test and finding objects
- [x] Unification of documentation for supported reports
- [x] Upgrade notes explaining the changes to raise awareness for those with references to Nessus in the pipelines 

fixes #7477 
[sc-466]

### CODE REVIEW

**Review by blakeaowens — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]
Thank you

File: docs/content/en/getting_started/upgrading.md [hunk 1]
```diff
@@ -67,6 +67,18 @@ godojo installations
 
 If you have installed DefectDojo on "iron" and wish to upgrade the installation, please see the [instructions in the repo](https://github.com/DefectDojo/godojo/blob/master/docs-and-scripts/upgrading.md).
 
+## Upgrading to DefectDojo Version 2.23.x.
+
+There is a migration from the legacy Nessus and Nessus WAS parsers to a single Tenable parser. The updated Tenable parser simply merges existing support for Nessus and Nessus WAS without introducing new functionality that could create instability
+
+There is a migration process built into the upgrade that will automatically convert exiting Nessus and Nessus WAS findings and tests into Tenables findings and tests
```
Comment (cneill) [inline docs/content/en/getting_started/upgrading.md:74]: Nit: s/Tenables/Tenable/

File: docs/content/en/getting_started/upgrading.md [hunk 2]
```diff
@@ -67,6 +67,18 @@ godojo installations
...
+
+**Breaking Change**
+
+If there is any use of the Nessus or Nessus WAS in automated fashion via the import and reimport API endpoints, the `scan-type` parameter needs to be udpated to `Tenable Scan`
```
Comment (cneill) [inline docs/content/en/getting_started/upgrading.md:78]: Nit: s/udpated/updated/

File: dojo/db_migrations/0187_nessus_to_tenable.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+# update the test type object as well as the scan type name
+def update_test(test, tenable_test_type) -> None:
+    if test.test_type.name in NESSUS_REFERENCES or test.scan_type in NESSUS_REFERENCES:
+        test.test_type = tenable_test_type
+        test.scan_type = tenable_test_type.name
+        test.save()
+
+
+# Update the found_by field to remove nessues/WAS and add tenable
```
Comment (cneill) [inline dojo/db_migrations/0187_nessus_to_tenable.py:18]: nit: s/nessues/nessus

**Review by cneill — APPROVED** [review]

**Pull request merged**